src/api/transfers/routes.js

```diff
@@ -26,7 +26,7 @@ const BaseJoi = require('joi-currency-code')(require('joi'))
 const Extension = require('joi-date-extensions')
 const Joi = BaseJoi.extend(Extension)
 const tags = ['api', 'transfers']
-const transferState = [ 'RECEIVED', 'RESERVED', 'COMMITTED', 'ABORTED', 'SETTLED' ]
+const transferState = ['RECEIVED', 'RESERVED', 'COMMITTED', 'ABORTED', 'SETTLED']
```

Comment:
I do think that these values should be loaded from the database instead of the hardcoding at some point but for now im happy with it